src/components/store/pages/callbacks/success.js

```diff
@@ -1,26 +1,52 @@
 import React, { useState, useEffect } from "react";
 import { useLocation } from "react-router-dom";
+import { faSpinner } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import Page from "../../page";
 
 export default () => {
   const query = new URLSearchParams(useLocation().search);
   const creditKeyId = query.get("id");
+  const [loading, setLoading] = useState(false);
   const [checkoutStatus, setCheckoutStatus] = useState(undefined);
 
   useEffect(() => {
     // complete order with backend
-    fetch(`https://c2rxvcaph7.execute-api.us-west-2.amazonaws.com/staging/demo-success?ckkey=${creditKeyId}`)
-      .then((res) => res.json())
-      .then((json) => {
-        setCheckoutStatus(json);
-      });
+    if (creditKeyId) {
+      setLoading(true);
+      fetch(`${process.env.REACT_APP_BACKEND_URL}${creditKeyId}`)
```

Comment:
so `fetch` doesn't yield an exception that can be caught as a failed promise when a request is anything besides a 2xx response.  Rather it provides an `ok` property on the response.  If you look at the `request.js` utlility in any of our other UIs, you can see what we do by wrapping `fetch` in an es6 style `Promise`.  I'd recommend doing something similar here, or even just bring `request.js` over and use it directly.